recipes-graphics/spark/spark_git.bb

```diff
@@ -0,0 +1,101 @@
+include pxcore.inc
+
+DEPENDS_append = " \
+    curl freetype util-linux libpng pxcore-libnode giflib sqlite3 \
+    wpeframework-tools-native wpeframework wpeframework-clientlibraries \
+"
+
+SRC_URI += " \
+    file://Spark.pc \
+    file://0001-nanosvg-patches.patch \
+    file://0002-pxScene-disable-2DMultisampleEXT.patch \
+    file://0003-spark-wpeframework-compositor.patch \
+    file://0004-dukluv-git.patch \
+    file://0005-pxScene-essos-support-for-shared-lib.patch \
+    file://0006-pxScene-initApp-load-issue-fix.patch \
+    file://0007-node-v10.15.3_mods.patch \
+"
+
+inherit cmake pkgconfig
+
+PACKAGECONFIG ?= " \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'wayland westeros', 'wpeframework', d)} \
+    turbojpeg \
+"
+
+PACKAGECONFIG[rtremote] = "-DBUILD_RTCORE_LIBS=ON -DBUILD_RTCORE_STATIC_LIB=OFF,,,rtcore rtremote"
+PACKAGECONFIG[wayland] = "-DBUILD_WITH_WAYLAND=ON -DPXCORE_WAYLAND_EGL=ON -DBUILD_PXSCENE_WAYLAND_EGL=ON,,wayland"
+PACKAGECONFIG[westeros] = "-DBUILD_WITH_WESTEROS=ON -DPXCORE_WAYLAND_EGL=ON -DBUILD_PXSCENE_WAYLAND_EGL=ON,,westeros"
+PACKAGECONFIG[wpeframework] = "-DBUILD_WITH_WPEFRAMEWORK=ON -DPXCORE_WPEFRAMEWORK=ON,,wpeframework"
+PACKAGECONFIG[turbojpeg] = "-DDISABLE_TURBO_JPEG=ON,,libjpeg-turbo"
```

Comment:
As this is disabling the TurboJPEG I would make it enabled by default like this:
```PACKAGECONFIG[turbojpeg] = ",-DDISABLE_TURBO_JPEG=ON,libjpeg-turbo"```
Then turbojpeg is not needed in the PACKAGECONFIG?=. If someone will need to disable the turbo JPEG support then it will be done by adding it to PACAKGECONFIG

Reply:
Ok, got it :)

Reply:
It will not add libjpeg-turbo in the build, even with disabled case also spark need libjpeg-turbo. So adding it in common DEPENDS list